### DISCUSSION TITLE
Frequency of addon release

### DISCUSSION THREAD

**Discussion by wrkn4alivn** [hassio-addons/addon-zwavejs2mqtt#131] [General]
Hi Frenck, thanks so maintaining so many addons and making it easy for users like me to bring valuable functionality to HA ecosystem!

I am curious what your overall strategy is for frequency of addon releases. I know that in some cases (like a prior release where proper notice of usage data sharing wasn't done) frequent releases can be undesirable. I also have a contrasting example where I'm still on v0.8.1, eagerly awaiting an update in the hopes that an issue in zwave-js has been resolved (https://github.com/home-assistant/core/issues/48605#issuecomment-821483052).

Do you kind of play it by ear, or have an approx number of days since the last update of the addon and/or zwavejs2mqtt? Should folks like me just sit tight, or do you welcome requests for a release? Thanks again!

**Comment by frenck** [comment]
Add-ons follow upstream releases as fast as possible, in this specific case I was awaiting a fix from upstream. This has been released this morning.

Asking for releases is generally completely ignored by me.

**Reply by wrkn4alivn** [reply]
Good clarification, much appreciated!

**Comment by jhein05** [comment]
Hi Frenck, 
just wondering if there is a reason that the addon is 3 releases behind. 
Thank you and your work is much appreciated!

**Reply by frenck** [reply]
Simply time and weekend. It has been crazy with the HA release and today I am catching up

**Reply by erwin-willems** [reply]
I think there is a dependency issue. I tried to update this add-on myself, but it looks like there is an issue with alpine 3.13+ to build zwavejs2mqtt. But the addon-base image is based on alpine 3.14.

See: https://github.com/zwave-js/zwavejs2mqtt/blob/master/docker/Dockerfile:
```
# STEP: 1 doens't work with alpine 3.13+
FROM node:16.3.0-alpine3.12 AS build-z2m
```

I would build with alpine 3.12 and then copy the files over to the addon-base image with:
```
COPY --from=build-container $src $dst
```

Happy to help with this :)

**Reply by erwin-willems** [reply]
I tried to rollback to an older issue, because I'm waiting for a fix for my Fibaro switches. These are currently not reporting the state from the first switch which is very annoying with 15+ modules and a lot of automations depending on these state (no pressure :) ). But the rollback failed, because the integration depends on the newer node-zwave-js v8.0.x codebase :(.

I'll try to be patient :)

**Reply by frenck** [reply]
There is no issue with this add-on on Alpine 3.14.

The Dockerfile linked is not used. As this application uses compiled code, I would never recommmend in grabbing it across from their base images, that is asking for problems

**Reply by erwin-willems** [reply]
I wasn't able to build the Docker image. But it's fine for now, I will wait until you release a new version :).

**Reply by jhein05** [reply]
> Simply time and weekend. It has been crazy with the HA release and today I am catching up

That sounds like a very good reason! Thanks man!
